source/includes/gcp/_firewall_rules.md

```diff
@@ -72,6 +73,7 @@ Retrieve a list of all firewall rules in a given [environment](#administration-e
 `selfLink`<br/>*string* | Server-defined URL for this resource.
 `sourceRanges`<br/>*Array[string]* | Server-defined URL for the resource.
 `targetTags`<br/>*Array[string]* | A list of tags that controls which instances the firewall rule applies to. If targetTags are specified, then the firewall rule applies only to instances in the VPC network that have one of those tags. If no targetTags are specified, the firewall rule applies to all instances on the specified network.
+`details`<br/>*string* | Concatenation of the IP protocols followd by the port ranges.
```

Comment:
Little typo here, should be followed

Reply:
1c902795ba00f6ba1f4af8f97b9112267ef3e8a7